app/code/Magento/Customer/Setup/CustomerSetup.php

```diff
@@ -127,11 +150,11 @@ public function getDefaultEntities()
     {
         $entities = [
             'customer' => [
-                'entity_type_id' => \Magento\Customer\Api\CustomerMetadataInterface::ATTRIBUTE_SET_ID_CUSTOMER,
-                'entity_model' => \Magento\Customer\Model\ResourceModel\Customer::class,
-                'attribute_model' => \Magento\Customer\Model\Attribute::class,
+                'entity_type_id' => CustomerMetadataInterface::ATTRIBUTE_SET_ID_CUSTOMER,
+                'entity_model' => Customer::class,
+                'attribute_model' => Attribute::class,
                 'table' => 'customer_entity',
-                'increment_model' => \Magento\Eav\Model\Entity\Increment\NumericValue::class,
+                'increment_model' => NumericValue::class,
```

Comment:
Also check below for `\Magento\Customer\Model\ResourceModel\Attribute\Collection::class`, please import it as well